tests/utils.py

```diff
@@ -26,7 +26,7 @@
 from dlt.common.schema.typing import TTableSchema
 from dlt.common.utils import set_working_dir
 
-from dlt.pipeline.exceptions import SqlClientNotAvailable
+from dlt.common.destination.exceptions import SqlClientNotAvailable
```

Comment:
this is ok because this is a part of tests! just remember that we need to be backward compatible with old dlt versions in the main code so you'd need to try both options there